### DISCUSSION TITLE
Is there a way to disable all live updates temporarily?

### DISCUSSION THREAD

**Discussion by taranlu-houzz** [Textualize/rich#1472] [Q&A]
Currently, I am trying use `pdb` via `breakpoint()`, but the various `rich` live display objects keep refreshing which makes it impossible to use an interactive debugger.

I am wondering if there is some sort of global env var escape hatch that could be used to disable all live updates?

**Comment by willmcgugan** [comment]
There isn't at the moment. However, there is a [related issue](https://github.com/willmcgugan/rich/issues/1465) I'll be looking at soon.
